apps/web/src/components/borrow/table/YourBorrowsTable.tsx

```diff
@@ -130,7 +139,7 @@ export const YourBorrowsTable: FunctionComponent = () => {
         titleComponent={
           <BorrowTitleBox
             isEmpty={userReservesWithDept.length === 0}
-            currentBorrowedValue={Number(currentBorrowedValue.toFixed(2))}
+            currentBorrowedValue={Number(currentBorrowedValue?.toFixed(2))}
```

Comment:
do you need to show 2 decimals if the value is `10.00` ? or just `10`?

you could create a function that will do something similar like `numberToPercentage` that we already have.


```tsx
// toFixed(2) will round it if there is more decimals 10.00001=>10.00
  //and toPrecision will reduce decimals if number is 10.00 to 10
Number(Number(value.toFixed(2)).toPrecision())
```

Reply:
Sure, this is a duplicate, we've already use numberToPercentage to calculate currentBorrowedValue, so I can remove this line Number(currentBorrowedValue?.toFixed(2)) and just pass currentBorrowedValue as it is